sample/ground_segmentation/scenario.ja.yaml

```diff
@@ -10,25 +10,8 @@ Evaluation:
     - sample_dataset:
         VehicleId: default
   Conditions:
-    Method: annotated_pcd # ground truthデータとして使用するデータの種類(annotated_rosbag : アノテーション済み点群が入ったinput_bag，annotated_pcd : アノテーション済みpcd.binファイル(t4_dataset/dataset/data/LIDAR_CONCAT/*.pcd.bin))
     ground_label: 6 # 地面点群のアノテーションラベル
     obstacle_label: 7 # 障害物点群のアノテーションラベル
```

Comment:
@nanoshimarobot 
as could you update `ground_label` and `obstacle_label` as a list for multiple classes selectioin as mentioned [here ](https://github.com/tier4/driving_log_replayer_v2/pull/156#issuecomment-3060809633)